### PR TITLE
Fix #3561 : Merged 2 version of profile-list-profile-view-xml into single xml

### LINKED ISSUE #3561: Merge profile_list_profile_view.xml into single xml file
Currently there are 2 versions of `profile_list_profile_view.xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

**Note**: In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.

### PR DESCRIPTION
**Pull request by Aakash1121**
## Explanation

Fixes #3561 : Merged portrait and landscape version profile-list-profile-view-xml of Edit-Profiles under Profile-Management.

## Essential Checklist

- [X] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [X] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [X] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [X] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [X] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [X] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

# Phone Before: 

Phone_portrait_before
Phone_landscape_before

<br>

# Phone After:

phone_portrait_after
phone_landscape_after

# Tablet Before:

tablet_landscape_before
tablet_portrait_before

<br>

# Tablet After:

tablet_portrait_after
tablet_landscape_after

### CODE REVIEW

**Comment by Aakash1121** [comment]
@rt4914 @yash10019coder , Can you let me know if I had solved the issue properly or not. 
And am I still assigned to this issue? as @rt4914 self-assigned him. 
Am a beginner, help me out with this.

**Comment by rt4914** [comment]
> @rt4914 @yash10019coder , Can you let me know if I had solved the issue properly or not. And am I still assigned to this issue? as @rt4914 self-assigned him. Am a beginner, help me out with this.

@Aakash1121 I will review it tonight.

File: app/src/main/res/layout/profile_list_profile_view.xml [hunk 1]
```diff
@@ -36,6 +36,7 @@
       android:layout_width="0dp"
       android:layout_height="wrap_content"
       android:layout_marginStart="20dp"
+      android:layout_marginBottom="10dp"
```
Comment (rt4914) [inline app/src/main/res/layout/profile_list_profile_view.xml]: Why was this value added ? It was not available earlier.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@Aakash1121 PTAL at suggestions. Thanks.

File: app/src/main/res/layout/profile_list_profile_view.xml (hunk 1 above)
Comment (Aakash1121) [reply app/src/main/res/layout/profile_list_profile_view.xml]: The profile image and profile name were not in symmetric order. So I added 10dp of marginBottom to the profile name textview.

Comment (Aakash1121) [reply app/src/main/res/layout/profile_list_profile_view.xml:39]: The profile image and profile name were not in symmetric order. So I added 10dp of marginBottom to the profile name textview.

**Comment by Aakash1121** [comment]
Hi @rt4914 , can you review the pull request. 
Also I have commented on the past review, please check it out.

**Comment by Aakash1121** [comment]
Hi @rt4914 , any updates on this issue. Do I have to do something further in this? Please let me know.

File: app/src/main/res/layout/profile_list_profile_view.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/layout/profile_list_profile_view.xml]: But that should be filed as a separate issue and then solved. As this PR is focused on merging of files only and therefore this change can create confusion. Also we generally do not write random values in xml file, we check them from our design files and from that we get the correct value and use that.

**Comment by rt4914** [comment]
@Aakash1121 Just replied here https://github.com/oppia/oppia-android/pull/4108#discussion_r791835072

File: app/src/main/res/layout/profile_list_profile_view.xml (hunk 1 above)
Comment (Aakash1121) [reply app/src/main/res/layout/profile_list_profile_view.xml]: Thanks @rt4914  for guiding, I will make the required changes.

**Comment by Aakash1121** [comment]
Hi @rt4914, I have made the required changes, can you review it.

**Comment by Aakash1121** [comment]
Hi @rt4914 , any updates for this issue?

**Comment by rt4914** [comment]
@Aakash1121 Please make sure that you reach out to your mentor @bkaur-bkj  so that review the PR first and also assign it to me for review when ready.

File: app/src/main/res/values-land/dimens.xml [hunk 2]
```diff
@@ -42,6 +42,9 @@
   <dimen name="profile_chooser_profile_view_margin_start">16dp</dimen>
   <dimen name="profile_chooser_profile_view_margin_end">16dp</dimen>
 
+  <!-- profile-list-profile-view -->
```
Comment (rt4914) [inline app/src/main/res/values-land/dimens.xml]: Rename to 
`<!-- Profile List Profile View -->`

Ditto everywhere else.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@Aakash1121 Thanks. Only one change suggested. Rest all is good.

File: app/src/main/res/values-land/dimens.xml (hunk 2 above)
Comment (Aakash1121) [reply app/src/main/res/values-land/dimens.xml]: Okay, will do the required changes.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Comment by Aakash1121** [comment]
Hi @rt4914 @bkaur-bkj , are there any steps left for merging the pull request?. Please let me know.

**Comment by bkaur-bkj** [comment]
> Hi @rt4914 @bkaur-bkj , are there any steps left for merging the pull request?. Please let me know.

HI @Aakash1121 Your PR is approved it will be merged by @rt4914

**Comment by bkaur-bkj** [comment]
@rt4914 PTAL

**Comment by Aakash1121** [comment]
Hi @rt4914 , can you merge this pull request, if it's proper?

**Comment by Aakash1121** [comment]
Hi @rt4914 @bkaur-bkj any updates for this pull request?

**Comment by rt4914** [comment]
@Aakash1121 Apologies for delay. Will approve this today.

**Pull request merged**